packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue

```diff
@@ -1,85 +1,103 @@
 <template>
-	<template v-if="config.isActive">
-		<section class="input-row">
-			<p>
-				Find the
-				<Dropdown
-					class="toolbar-button"
-					v-model="knobs.optimizationType"
-					:options="OPTIMIZATION_TYPES"
+	<div class="policy-group">
+		<div class="form-header">
+			<div>
+				<tera-input v-if="isEditing" v-model="knobs.name" placeholder="Policy bounds" />
+				<h6 v-else>{{ knobs.name }}</h6>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
 				/>
```

Comment:
See https://github.com/DARPA-ASKEM/terarium/pull/4080/files#r1670837039

Reply:
Thank you for this!